utils/watcher_strace.py

```diff
@@ -107,6 +113,20 @@
                                         item)
                                     break
                 # For file access, put it here to ignore sendmsg
+                # For music file being set to play
+                elif 'setMedia()' in output:
```

Comment:
I think you can move this part to group with line 68

First check if it's a write event,
Check if it's the desired event setMedia()
    Yes -> call printer
    No -> continue the loop